deflate.c

```diff
@@ -71,14 +71,15 @@ const char PREFIX(deflate_copyright)[] = " deflate 1.3.1 Copyright 1995-2024 Jea
  */
 #ifdef S390_DFLTCC_DEFLATE
 #  include "arch/s390/dfltcc_deflate.h"
+#  define PAD_WINDOW            PAD_4096
+#  define WINDOW_PAD_SIZE       4096
```

Comment:
Why is S390 pad size so high? Maybe a comment about it?

Reply:
Window needs to be page-aligned for DFLTCC instruction it seems.
A comment is a good idea.

Reply:
I can confirm that window needs to be page-aligned, so 4k is the right value.